cosmoz-image-viewer.html

```diff
@@ -124,6 +125,7 @@
 				justify-content: center;
 				background-color: black;
 				pointer-events: none;
+				height: 600px;
```

Comment:
Hmm where did this come from? We can keep it if needed but why didn't we need it before?